File: example/optimizely/ios/sample-OPT.m
/// Create visitor
FFVisitor * visitor1 = [[[[FeatureFlag sharedInstance] newVisitor:@"visitor_1" instanceType:InstanceSHARED_INSTANCE] withContextWithContext:@{@"age":@18} ] build];

/// Fetch flags
[visitor1 fetchFlagsOnFetchCompleted:^{
    flag = [visitor1 getFeatureVariableString:@"featureKey", variableKey:@"OPT-flag-m", userId:@"user_123", attributes:attributes];
}];

File: cmd/analyze/example/optimizely/ios/sample-OPT.swift
// Create the visitor
let visitor1 = FeatureFlag.sharedInstance.newVisitor("visitor_1").build()

// Fetch flags
visitor1.fetchFlags {

    let flag = visitor1.getFeatureVariableString(featureKey:"my_feature_key", variableKey:"OPT-flag-swift", userId:"user_123", attributes:attributes)

}